options/options.go

```diff
@@ -156,9 +157,12 @@ func (o *Options) Initialize() error {
 	// o.EnableLicense = "1"
 	// o.EnableDangerousWorkflow = "1"
 
-	_, tokenSet := os.LookupEnv(EnvGithubAuthToken)
-	if !tokenSet {
-		inputToken := os.Getenv(EnvInputRepoToken)
+	inputToken := os.Getenv(EnvInputRepoToken)
+	if inputToken == "" {
+		inputToken = os.Getenv(EnvInputInternalRepoToken)
+		if inputToken == "" {
+			return errInternalDefaultTokenEmpty
+		}
```

Comment:
Verification of the token value should move to `Validate` function.

Reply:
already done. you're too fast :-)